From d19cab13f77a3366bbb1c069e5a6410a56bbe1cd Mon Sep 17 00:00:00 2001
From: Robert Nelson <robertcnelson@gmail.com>
Date: Tue, 23 Jun 2020 09:33:13 -0500
Subject: [PATCH] bits to bitops

Signed-off-by: Robert Nelson <robertcnelson@gmail.com>
---
 mikrobus_manifest.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/mikrobus_manifest.h b/mikrobus_manifest.h
index 26ae7a4..b82bc0f 100755
--- a/mikrobus_manifest.h
+++ b/mikrobus_manifest.h
@@ -7,7 +7,7 @@
 #ifndef __MIKROBUS_MANIFEST_H
 #define __MIKROBUS_MANIFEST_H
 
-#include <linux/bits.h>
+#include <linux/bitops.h>
 #include <linux/types.h>
 #include <linux/property.h>
 
@@ -125,4 +125,4 @@ struct mikrobus_manifest
 bool mikrobus_manifest_parse(struct click_board_info *info, void *data, size_t size);
 size_t mikrobus_manifest_header_validate(void *data, size_t size);
 
-#endif /* __MIKROBUS_MANIFEST_H */
\ No newline at end of file
+#endif /* __MIKROBUS_MANIFEST_H */
-- 
2.27.0

